CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+
+#### What are Rearc's goals for ADX?
+We at Rearc are working tirelessly to lend greater accessibility to interesting and/or important datasets across various disciplines and sources. We realize the direct integration of the ADX, along with other AWS services, facilitates a convenient manner for our subscribers to consume data. For data providers we can supply an automation pipeline, leveraging the AWS platform, to ensure the ubiquity of your data for your consumers.
+
+#### What is Rearc's philosophy towards dataset formats?
+We try as much as possible to preserve the integrity of data we provide through ADX, and most of the time this means delivering datasets exactly as they were presented from their source. Sometimes we make minor alterations to datasets to provide wider usability for ADX subscribers (e.g. adjusting CSV files for SQL column naming conventions). For situations where we are unable to maintain the original data file format (often due to a request from the data consumer or a discrepancy in a source's delivery pipeline), we try to limit the extent of needed transformations as much as possible.
+
+#### What tools are you using throughout your ADX Products?
+- On our local devices we have chosen to use a combination of [Python 3](https://www.python.org), [Pip](https://pypi.org/project/pip/), [JQ](https://stedolan.github.io/jq/), [AWS CLI V2](https://aws.amazon.com/cli/) and other related developer tools. The specific tools utilized varies on a project-by-project basis. The Python portions of our source code can be adapted into another language with a supported AWS Lambda runtime.
+- Most of our ADX projects limit the usage of packages to [The Python Standard Library](https://docs.python.org/3.7/library/index.html). By default, Python provides a robust set of tools to access and manipulate popular data file formats, and we exploit these features to their fullest extent. Frequent packages we used when interacting with datasets include [os](https://docs.python.org/3.7/library/os.html), [urllib](https://docs.python.org/3.7/library/urllib.html), [json](https://docs.python.org/3.7/library/json.html) and [csv](https://docs.python.org/3.7/library/csv.html). We also utilize [Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html), AWS's ADK for Python, which is automatically bundled into AWS Lambda's Python runtimes.
+- We are creating various AWS resources throughout the process of deploying and maintaining our ADX projects, including [Data Exchange](https://docs.aws.amazon.com/data-exchange/), [S3](https://docs.aws.amazon.com/s3/), [Lambda](https://docs.aws.amazon.com/lambda/) and [EventBridge](https://docs.aws.amazon.com/eventbridge/). The creation of most of the needed resources is streamlined by the [CloudFormation](https://docs.aws.amazon.com/cloudformation/) template included with this project, ([pre-processing-cfn.yaml](./pre-processing/pre-processing-cfn.yaml)).
```

Comment:
I think this whole section can be cut down to we use Python 3, CloudFormation, and Lambda roughly.

Reply:
Cut down "What tools are you using throughout your ADX Products?" section to just Python 3, CloudFormation, and Lambda. Followed up with a brief bullet stating that the exact tools used vary depending on the specific dataset being published